src/client/app/reducers/formReducer.js

```diff
@@ -0,0 +1,68 @@
+import {
+    FORM_SUCCESS,
+    SUBMIT_ERROR,
+    SET_FORM_FIELDS,
+    UPDATE_FORM_FIELD,
+    SET_USER_AUTOCOMPLETE
+} from '../constants';
+
+const initialstate = {
+    errors: []
+};
+
+export default function formReducer (state = initialstate, action) {
+    switch (action.type) {
+
+    case SET_FORM_FIELDS:
+        const { fields, modelName, formType } = action.data;
+        return {
+            ...state,
```

Comment:
Nope, async/await and Object spread are 100% compatible, if that's what you mean! So, should I add browser compatibility to the readme or just that we're using some ES7 features? I've stayed away from decorators because Babel 6 doesn't support them natively any more. Well, they'll probably get added back since they're likely to be adopted in ES7, but who knows... it's JS right?